developer-docs/installation/configuring_userprofile.md

```diff
@@ -9,6 +9,54 @@ published: true
 
 ## Overview
 
+Sunbird Release 1.11, supports user profile configuration through the system_settings table in Cassandra.
+
 ## Prerequisites
 
-## Configuring User Profile 
\ No newline at end of file
+1. Access to cassandra database
+
+## Configuring User Profile 
+
+During cassandra migration for Release 1.11 (CQL file version 1.25) will configure the following user profile configuration in system_settings table.
+
+```
+{
+  "fields": [
+    "firstName",
+    "lastName",
+    "profileSummary",
+    "avatar",
+    "countryCode",
+    "dob",
+    "email",
+    "gender",
+    "grade",
+    "language",
+    "location",
+    "phone",
+    "subject",
+    "userName",
+    "webPages",
+    "jobProfile",
+    "address",
+    "education",
+    "skills",
+    "badgeAssertions"
+  ],
+  "publicFields": [
+    "firstName",
+    "lastName",
+    "profileSummary"
+  ],
+  "privateFields": [
+    "email",
+    "phone"
+  ]
+}
+```
+

```

Comment:
@JaikumarSRajan Is this part of any API? or is it just code to update the system settings table? The table below does not add much value as the description is too generic. There needs to be clarity on this.

Reply:
@Sunitashelar The JSON snippet is the configuration value which goes into cassandra database - system_settings table. The table explains each key fields in the JSON snippet so that the adopter may choose to change the values accordingly.

Reply:
Made changes as per comments